frontend/src/pages/account-settings/account-settings.tsx

```diff
@@ -71,10 +71,22 @@ const AccountSettings: React.FC = () => {
           <div className={styles['container']}>
             <div className={styles['page-title']}>Account Settings</div>
             <div className={styles['page-nav']}>
-              <Link to={AppRoute.PROFILE} className={styles['profile-link']}>
+              <Link
+                to={AppRoute.PROFILE}
+                className={getValidClassNames(
```

Comment:
fix it by adding the `activeClassName` prop

Reply:
there is no link component to use activeClassName. It seems to me then this option is more correct.
there is not a Link but a NavLink
![image](https://user-images.githubusercontent.com/84984785/230603835-54fe1fb0-b118-4854-b9a3-c9a7ef6f7a78.png)